app/views/Project/Tagging/Export/ExportHistory/index.tsx

```diff
@@ -113,8 +110,6 @@ function ExportHistory(props: Props) {
     const [activePage, setActivePage] = useState(1);
     const [exportedAt, setExportedAt] = useState<DateRangeValue>();
     const [searchText, setSearchText] = useState<string>();
-    // FIXME: we can safely remove useDebouncedValue if we are using TextInput from deep-ui
-    const debouncedSearchText = useDebouncedValue(searchText, debounceTime);
```

Comment:
Any reason we removed debouncing?

Reply:
The TextInput component supports debouncing